src/natcap/invest/ui/model.py

```diff
@@ -1069,15 +1069,15 @@ def post_warnings(self, validation_warnings):
         self.warnings = validation_warnings
 
         if validation_warnings:
-            # cgi.escape handles escaping of characters <, >, &, " for HTML.
+            # html.escape handles escaping of characters <, >, &, " for HTML.
             self.title_label.setText(
                 '<h2>Validation warnings found</h2>'
                 '<h4>To ensure the model works as expected, please fix these '
                 'erorrs:</h4>')
             self.label.setText(
                 '<ul>%s</ul>' % ''.join(
                     ['<li><b>%s</b>: %s</li>' % (
-                        ', '.join(labels), cgi.escape(warning_, quote=True))
+                        ', '.join(labels), html.escape(warning_))
```

Comment:
Ran into this issue a little while ago, fixing it in this PR. `cgi.escape` has been deprecated since 3.2 and doesn't exist in 3.8 (https://docs.python.org/3.7/library/cgi.html#cgi.escape). `html.escape` now defaults to `quote=True`.